### ISSUE TITLE
backup failed!

### ISSUE THREAD

**Issue by mostafa6765** [JackieDo/Laravel-Dotenv-Editor#18]
`php artisan dotenv:get-backups`

```php
[ReflectionException]
  Method Jackiedo\DotenvEditor\Console\Commands\DotenvGetBackupsCommand::handle() does not exist
```
@JackieDo

**Comment by mostafa6765** [comment]
See this @JackieDo  
https://github.com/laravel/framework/commit/ac9f29da4785c8b942ced8c4ab308214d618a22b
may be this make error

**Comment by dacoto** [comment]
Solved: @mostafa6765 @deleugpn @JackieDo Pull request: #23

**Comment by JackieDo** [comment]
Fixed this bug on release [1.0.8](https://github.com/JackieDo/Laravel-Dotenv-Editor/releases/tag/1.0.8)
